napari/benchmarks/benchmark_qt_viewer_labels.py

```diff
@@ -83,3 +86,65 @@ def time_fill(self):
     def time_on_mouse_move(self):
         """Time to drag paint on mouse move."""
         self.viewer.window._qt_viewer.canvas._on_mouse_move(self.event)
+
+
+class LabelRenderingSuite:
+    """Benchmarks for rendering the Labels layer."""
+
+    param_names = ["radius", "dtype"]
+    params = ([20, 400, 2000], [np.uint8, np.uint16, np.uint32])
+    if "PR" in os.environ:
+        skip_params = Skiper(lambda x: x[0] >= 100)
```

Comment:
holy shit I didn't know this was valid syntax 🤯 

Skiper is a misspelling, btw, should be Skipper I guess? Not for this PR, I just hadn't seen that class. I'm also not 100% about skipping things, since the benchmarks only run when asked anyway? (And they don't seem to take all that long...)

Reply:
You are even a person who accepts PR and merges it: https://github.com/napari/napari/pull/6278

Currently, benchmarks for small parameters are run on every PR. 

https://github.com/napari/napari/blob/93af19af93da596ff9c5e595a72af8eac8bfb0e9/.github/workflows/test_pull_requests.yml#L168-L193

This is to not accidentally break benchmarks.

Full benchmarks are a few hours. 

This is not fully valid syntax (it is not documented), but the code uses `in` operator and I use this hack for cleaner code.